src/main/java/com/example/homework10/mapper/MusicMapper.java

```diff
@@ -0,0 +1,27 @@
+package com.example.homework10.mapper;
+
+import com.example.homework10.entity.Music;
+import org.apache.ibatis.annotations.*;
+
+import java.util.List;
+
+@Mapper
+
+public interface MusicMapper {
+    @Select("SELECT * FROM music")
+    List<Music> findAll();
+
+    @Select("SELECT * FROM music WHERE id = #{id}")
+    Music findById(int id);
+
+    @Insert("Insert INTO music (title, singer) VALUES (#{title}, #{singer})")
```

Comment:
SQLの予約語は大文字で統一してほしいです。
```suggestion
    @Insert("INSERT INTO music (title, singer) VALUES (#{title}, #{singer})")
```

Reply:
[1a1cf5f](https://github.com/yuu3415/homework10/pull/3/commits/1a1cf5f9f5ee3eed143df28a7d1f78c1249f3742)
申し訳ございません。
直し忘れが多数ありましたので修正いたしました。